docs/source/tutorial/models_examples.rst

```diff
@@ -0,0 +1,23 @@
+Models Client Examples
+======================
+
+The ``ModelsClient`` class encapsulates interactions with
+trained models on Citrination.
+
+Predict
+-------
+
+The ``.predict()`` method allows you to make a prediction with
+a model by referencing the ID of a Data View.
+
+Custom Predict
```

Comment:
This doesn't need to be included in the tutorial.

Reply:
👍